content/id/docs/contribute/localization_id.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Dokumentasi Khusus Untuk Translasi Bahasa Indonesia
+content_type: concept
+---
+
+<!-- overview -->
+
+Panduan khusus untuk bergabung ke komunitas SIG DOC Indonesia dan melakukan 
+kontribusi untuk mentranslasikan dokumentasi Kubernetes ke dalam Bahasa 
```

Comment:
```suggestion
kontribusi untuk menerjemahkan dokumentasi Kubernetes ke dalam Bahasa 
```

Reply:
let's not use "translasi" for the rest of the page, same comment for the rest of occurence in this page

Reply:
Agree!